### DISCUSSION TITLE
Site Migration

### DISCUSSION THREAD

**Discussion by aswin2411** [kimai/kimai#3176] [On-Premise: Installation & Updates]
Hi Team, 

I am very happy to use Kimai and it perfectly suits our requirement. 
we have developed site in a server and planning to move to another server. 
Does that work if I change the DB details ? What are the other configurations I need to do ?

Thanks 
Aswin

**Comment by kevinpapst** [comment]
You can install Kimai on the new server, import your database dump from the old server, change your .env to point to the imported database and run `bin/console kimai:update` as last step.

Backup and restore is basically the same as moving to a new server:
https://www.kimai.org/documentation/backups.html
